apps/bfd-pipeline/bfd-pipeline-shared-utils/src/main/java/gov/cms/bfd/pipeline/sharedutils/PipelineApplicationState.java

```diff
@@ -94,7 +95,7 @@ public PipelineApplicationState(
    * Create pooled data source used to communicate with the database.
    *
    * @param dataSourceFactory the {@link DataSourceFactory} to use for the application's DB (which
-   *     this will use to create {@link #getPooledDataSource()})
```

Comment:
These various links should still work fine when you build and the generated classes get made, right? I'm unsure why theyre all being removed.

Reply:
Compile fails when generating javadoc if they are in.  Might be a workaround for that, I'll check

Reply:
What I have found is you can {@link #fieldName} rather than the getter method.